adapters/mock-da/src/types/mod.rs

```diff
@@ -140,6 +140,7 @@ pub struct MockBlob {
     pub(crate) hash: [u8; 32],
     /// Actual data from the blob. Public for testing purposes.
     pub data: CountedBufReader<Bytes>,
+    pub(crate) zk_proofs_data: Vec<u8>,
```

Comment:
Let's add a small doc to describe the attribute? Also, as optional nit, if this is the concrete proof serialized, we should probably name it `zk_proofs`, as `zk_proofs_data` might give the impression we are storing proof metadata, such as witnesses only

Reply:
Thx, added a comment it. The `zk_proofs_data = proof + public input`